### PR TITLE
Added support for DescribeStreamSummary

### PR DESCRIPTION
**Pull request by Hulios**
Added support for [DescribeStreamSummary](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DescribeStreamSummary.html)

This should solve the issue from localstack https://github.com/localstack/localstack/issues/1553

### CODE REVIEW

**Comment by Hulios** [comment]
@mhart Request for a review.

File: test/getRecords.js [hunk 1]
```diff
@@ -302,73 +302,75 @@ describe('getRecords', function() {
...
+                    },
+                  ],
+                })
+
+                done()
+              })
             })
           })
-        })
+        }, 1)
```
Comment (mhart) [inline test/getRecords.js]: Can you explain this change? It seems like a hack of some sort

Comment (Hulios) [reply test/getRecords.js]: @mhart Flow of this test looks like this:
request `PutRecords` with `records1` at `time1` 
request `PutRecords` with `records2` at `time2` (called `secondInsertSeconds`)
request `GetShardIterator` with `secondInsertSeconds` and type `AT_TIMESTAMP`

Sometimes, when `time1 == time2` this test fails as records from both inserts are returned. Example: https://travis-ci.org/mhart/kinesalite/builds/586195126

Thats why I have introduced minimal delay to make sure that `time1 != time2`

**Comment by mhart** [comment]
Gotcha, makes sense on the test – thanks for fixing that. I modified it slightly in https://github.com/mhart/kinesalite/commit/d0a27d95b5ffaa5424569a37b57ca22465e3305a

Pulled your other commit into master (and made a slight addition to include `ConsumerCount`) – published as v3.2.0 – thanks for the addition!

**Pull request closed without merging**